cfgov/prepaid_agreements/jinja2tags.py

```diff
@@ -14,8 +14,9 @@ def remove_url_parameter(request, discards):
     """
     query = request.GET.copy()
```

Comment:
I think I'd like to see a lot of this refactored to use object methods, while we're here, so that lines 15-19 become:

```python
params = dict(request.GET.lists())
items = {}
for key in params.keys():
```

I'm going to try the suggestion thing here, per-line... I dunno if it'll work well.

(My kingdom for multi-line suggestions in GitHub — you can choose to commit these and squash them or just re-write them all in one commit if they make sense to you.)

```suggestion
```